pkg/kubectl/cmd/top_node.go

```diff
@@ -82,16 +83,22 @@ func NewCmdTopNode(f cmdutil.Factory, out io.Writer) *cobra.Command {
 			if err := options.Complete(f, cmd, args, out); err != nil {
 				cmdutil.CheckErr(err)
 			}
+			sorting, err := cmd.Flags().GetString("sort-by")
```

Comment:
I don't think you need to read that again manually. You've already assinged `SortBy` member to `sort-by` flag.